src/cmd-compress

```diff
@@ -29,8 +29,16 @@ parser.add_argument("--compressor",
                     choices=['xz', 'gzip'],
                     default=DEFAULT_COMPRESSOR,
                     help=f"Compressor to use, default is {DEFAULT_COMPRESSOR}")
+parser.add_argument("--fast", action='store_true',
+                    help="Compress with gzip -1")
 args = parser.parse_args()
 
+# that's the tool default
+gzip_level = 6
```

Comment:
Shouldn't we default to 9 for release artifacts?

Reply:
Probably, I think we were just letting gzip default to level 6. Let's do that in a separate PR though? I think it's worth higher visibility than just another commit buried here.

Reply:
https://github.com/coreos/coreos-assembler/pull/1713